SQLitePCLRaw.bundle_sqlite3mc/SQLitePCLRaw.bundle_sqlite3mc.csproj

```diff
@@ -5,13 +5,13 @@
     <AssemblyName>SQLitePCLRaw.batteries_v2</AssemblyName>
     <RootNamespace>SQLitePCL</RootNamespace>
     <PackageId>$(MSBuildProjectName)</PackageId>
-    <Description>A SQLitePCLRaw bundle for SQLite3 Multiple Ciphers--an encryption extension for SQLite with support for multiple ciphers. Includes SQLite3 Multiple Ciphers $(Sqlite3MCVersion), based on SQLite $(SQLiteVersion). NOTICE! This package is maintained separately from the SQLitePCLRaw project.</Description>
```

Comment:
For context, I originally added this notice to avoid any confusion that might arise by using package names that were so similar to the SQLitePCLRaw-maintained ones.